openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -218,8 +218,12 @@ class LibraryXBlockMetadata:
     modified = attr.ib(type=datetime)
     display_name = attr.ib("")
     last_published = attr.ib(default=None, type=datetime)
+    last_draft_created = attr.ib(default=None, type=datetime)
+    last_draft_created_by = attr.ib(default=None, type=datetime)
```

Comment:
@rpenido I think this is string

```suggestion
    last_draft_created_by = attr.ib("")
```

Reply:
Good catch! Fixed!